index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+	<title>Apple's Clone</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+<!--navbar: -->
+    <nav class="nav-bar-apple">
+        <ul>
+            <li><a><img src="images/logo.png" alt="apple-logo"></a></li>
+            <li><a href="#">Store</a></li>
+            <li><a href="#">Mac</a></li>
+            <li><a href="#">iPod</a></li>
+            <li><a href="#">iPhone</a></li>
+            <li><a href="#">iPad</a></li>
+            <li><a href="#">iTunes</a></li>
+            <li><a href="#">Support</a></li>
+            <li><input id="search" type="text"></li>
+        </ul>          
+    </nav>
+    
+<!--grid with five background-images on CSS: -->
+    <div class="all-pictures">
+        <div class="bigger-fig"></div>
+        <div class="medium-fig-2"></div>
+        <div class="medium-fig-3"></div>
+        <div class="medium-fig-4"></div>
+        <div class="medium-fig-5"></div>
+    </div>
+<!--images over the background-images: -->
+    <div class="phrases">
+        <div class="first-phrase">
+            <a href="#"><img src="images/first-phrase.png" alt="phrase"></a>
+        </div>
```

Comment:
`phrases` section:
- [ ] Each link is an `<a>` HTML tag, with a `<div>` tag inside it.
- [ ] The `<div>` tag  inside the `<a>` tag should use an image as background -- instead of `<img>` tag, [img tag vs background-image](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)